### PR TITLE
make user_data a list

### PR DESCRIPTION
**Pull request by SamClinckspoor**
this allows different user_data for each instance when
instance_count > 1

breaking changes: user_data is now a list type

### CODE REVIEW

File: bastion/variables.tf [hunk 1]
```diff
@@ -74,5 +74,5 @@ variable "root_vl_delete" {
 }
 
 variable "user_data" {
-  default = ""
+  default = [""]
```
Comment (venturel) [inline bastion/variables.tf]: shouldn't this be of type list?

**Comment by iuriaranda** [comment]
if we have multiple instances, we need to explicitly add a user data for each of them?

**Comment by SamClinckspoor** [comment]
Yes, I don't think there is a way around that

**Comment by iuriaranda** [comment]
Well, it's a bit inconvenient but yes, I don't see another option neither.

**Pull request merged**